### ISSUE TITLE
Line breaks within logged messages

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#238]
The disadvantage of using explicit line breaks within logged messages is that lines after the line break does not start with a timestamp.

The line (in CSetup.py)

`logger.warn(f"Running with configuration level 4! \nSelected configuration file '{RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestCfgFile}'")`

causes:

```
20231212 12:05:34.548 - WARN - Running with configuration level 4! 
Selected configuration file 'D:/RobotFramework/python39/lib/site-packages/RobotFramework_TestsuitesManagement/Config/robot_config.jsonp'
```

In my opinion this can be avoided.

Another aspect is the better usage of the range of available trace levels. I ask myself if the selected configuration file really should be part of the warning. In my opinion this is a simple INFO. And the configuration file is 'loaded' (and not 'selected'), because in especially this case the user has not selected anything.

My proposal is the following alternative version:

```
logger.warn(f"Running with configuration level 4!")
logger.info(f"Loaded configuration file '{RobotFramework_TestsuitesManagement.CTestsuitesCfg.oConfig.sTestCfgFile}'")
```

This causes (looking much more plausible):

```
20231212 13:04:34.376 - WARN - Running with configuration level 4!
20231212 13:04:34.377 - INFO - Loaded configuration file 'D:/RobotFramework/python39/lib/site-packages/RobotFramework_TestsuitesManagement/Config/robot_config.jsonp'
```
Some lines later in the log file the configuration file is logged again (also as INFO):

`20231212 13:04:34.377 - INFO - CfgFile Path: 'D:/RobotFramework/python39/lib/site-packages/RobotFramework_TestsuitesManagement/Config/robot_config.jsonp'`

This is redundant. And the abbreviation 'CfgFile Path' is hard to capture for a reader.

I would prefer to have the information about the loaded configuration file only once, in a line like this:

`INFO - Loaded configuration file '.....`

**Comment by test-fullautomation** [comment]
Hi Son,
please implement according to Holgers proposal.
Thank you,
Thomas

**Comment by namsonx** [comment]
Hello Thomas,

I updated as Holger's proposal.

Thank you,
Son

**Comment by HolQue** [comment]
Retest successful. Issue can be closed.

**Comment by test-fullautomation** [comment]
Released with RobotFramework AIO 0.12.0
